test/test_apprise_config.py

```diff
@@ -156,10 +148,6 @@ def test_apprise_config(tmpdir):
         # Iñtërnâtiônàlization Testing
         windows://"""
 
-    if six.PY2:
-        # decode string into unicode
-        istr = istr.decode('utf-8')
-
     # Write our content to our file
     t = tmpdir.mkdir("internationalization").join("apprise")
     with io.open(str(t), 'wb') as f:
```

Comment:
In `ConfigFile.py`, there is a comment about using `io.open()`:
> Python 3 just supports `open()`, however to remain compatible with Python 2, we use the `io` module.

I think the corresponding occurrences of `io.open()` (9 pcs.) also may be revisited with a subsequent patch?

Reply:
Good catch!... testing probably mocks `io` too and will have to be updated to use `builtins`

Reply:
If you're in the mood, I will be happy if you can take it and add it to the branch. I will add my other adjustments (fixing a few typos and `noqa`s, and solving the `cryptography<3.4` pinning) later.

Reply:
No worries; done! :+1:

Reply:
That was quick! Thank you for 2ec6f5f.